docs/src/documenter.md

```diff
@@ -5,7 +5,7 @@ and notebooks. Typically, though, these files will be used to render documentati
 for your package. The generators ([`Literate.markdown`](@ref), [`Literate.notebook`](@ref)
 and [`Literate.script`](@ref)) supports a keyword argument `documenter` that lets
 the generator perform some extra things, keeping in mind that the source code have been
```

Comment:
Also, "mind" sounds redundant in this sentence:

> keeping in mind that the source code have been written with Documenter.jl in mind

But I don't really have a suggestion :)

Reply:
Yea, I will try to reword it sometime.